runner/ansible/roles/2.1.3/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '2.1.3'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # SAP HANA Resource Agent is not configured, then pass
+    cibadmin -Q --xpath "//primitive[@type='SAPHana']/@type" || exit 0
```

Comment:
This should report a `exit 1`. If this primitive doesn't exist we have a missconfigured cluster.
(Update the comment above too)

Reply:
fixed